docs/development/release.md

```diff
@@ -99,8 +93,73 @@ export RELEASE_BRANCH=release-${RELEASE_MAJOR_MINOR_VERSION}
    - As the pull request target branch, pick the proper destination branch from the first step of this section.
     
 1. Merge the pull request.
+
+### Prepare Hub for release 
+
+1. Checkout the [hub-manifests](https://github.com/capactio/hub-manifests) repository destination branch for the pull request.
+
+    - For major and minor release versions, set the destination branch to `main`. 
+    - For patch releases, set the destination to corresponding release branch. For example, for `0.3.1` release, checkout the `release-0.3` branch.
+
+    ```bash
+    git checkout {destination-branch}
+    ```
+
+1. Create and checkout new branch:
     
-### Create release branch
+   ```bash
+   git checkout -b prepare-${RELEASE_VERSION}
+   ```   
+
+1. Replace all Runners images in [`manifests/implementation/runner`](https://github.com/capactio/hub-manifests/tree/main/manifests/implementation/runner) to release versions.
+   
+1. Review and commit the changes:
+
+   ```bash
+   git add .
+   git commit -m "Update Runners images to release versions"
+   ```
+    
+1. Create the pull request from the branch.
+   
+   - In the pull request description, write the GitHub release notes that will be posted with the release to review.
+   - As the pull request target branch, pick the proper destination branch from the first step of this section.
+    
+1. Merge the pull request.
+
+#### Create release branch
+
+If you release major or minor version, create a dedicated release branch.
+
+1. Checkout the destination branch and pull the latest changes:
+
+    ```bash
+    git checkout {destination_branch}
+    git pull
+    ```
+
+1. Create a new branch:
+   
+    ```bash
+    git checkout -b ${RELEASE_BRANCH}
+    ```
+
+1. Create new tag and push it and release branch to upstream:
+
+    > **NOTE**: Git tag is in a form of SemVer version with `v` prefix, such as `v0.3.0`.
+   
+    ```bash
+    git tag v${RELEASE_VERSION} HEAD
+    ```
+   
```

Comment:
This is also something we should remove, right?

There's no such tag on our `hub-manifests` repo, and that's fine for me, as the manifests are versioned separately and there won't be any breaking changes on release branches.

```suggestion
```